airflow/ui/src/components/ParseDag.tsx

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Button } from "@chakra-ui/react";
+import { useQueryClient } from "@tanstack/react-query";
+import { FiRefreshCw } from "react-icons/fi";
+
+import {
+  useDagParsingServiceReparseDagFile,
+  UseDagServiceGetDagDetailsKeyFn,
+  UseDagSourceServiceGetDagSourceKeyFn,
+} from "openapi/queries";
+
+type Props = {
+  readonly dagId: string;
+  readonly fileToken: string;
+};
+
+const ParseDag = ({ dagId, fileToken }: Props) => {
+  const queryClient = useQueryClient();
+
+  const onSuccess = async () => {
+    await queryClient.invalidateQueries({
+      queryKey: UseDagServiceGetDagDetailsKeyFn({ dagId }),
+    });
+
+    await queryClient.invalidateQueries({
+      queryKey: UseDagSourceServiceGetDagSourceKeyFn({ dagId }),
+    });
+  };
+
+  const { mutate } = useDagParsingServiceReparseDagFile({ onSuccess });
+
+  return (
+    <Button
+      borderColor="border.emphasized"
+      onClick={() => mutate({ fileToken })}
+      variant="ghost"
```

Comment:
I think we want `variant="outline"`

Reply:
Thanks, used `outline` and removed the `borderColor` attribute.